features/bevragen/persoon/verblijfplaats/dev/autorisatie-gba.feature

```diff
@@ -0,0 +1,300 @@
+      | gemeenteCode | 0599   |
+      Als gba personen wordt gezocht met de volgende parameters
+      | naam                | waarde                                                                                             |
+      | type                | RaadpleegMetBurgerservicenummer                                                                    |
+      | burgerservicenummer | 000000024                                                                                          |
+      | fields              | burgerservicenummer,verblijfplaats.verblijfadres.korteNaam,verblijfplaats.verblijfadres.huisnummer |
+      Dan heeft de response een persoon met de volgende 'verblijfplaats' gegevens
+      | naam       | waarde          |
+      | straat     | Borgesiusstraat |
+      | huisnummer | 103             |
+
+
+  Rule: Wanneer met fields een veld van verblijfplaats wordt gevraagd en de afnemer is niet geautoriseerd voor een van de velden waarmee type verblijfplaats wordt bepaald, wordt een foutmelding gegeven
+    Om een veld van verblijfplaats te mogen vragen moet de afnemer geautoriseerd zijn voor alle LO BRP rubrieken waar het verblijfplaatstype mee bepaald wordt:
+    - 81110: straatnaam
+    - 81210: locatiebeschrijving
+    - 81310: land
+
+    @fout-case
+    Abstract Scenario: Afnemer is wel geautoriseerd voor gevraagde veld, maar niet voor alle velden nodig voor verblijfplaats type
+      Gegeven de afnemer met indicatie '12345' heeft de volgende 'autorisatie' gegevens
+      | Rubrieknummer ad hoc (35.95.60) | Medium ad hoc (35.95.67) | Datum ingang (35.99.98) |
+      | <ad hoc rubrieken>              | N                        | 20201128                |
+      En de geauthenticeerde consumer heeft de volgende 'claim' gegevens
+      | naam         | waarde |
+      | afnemerID    | 12345  |
+      | gemeenteCode | 0518   |
+      Als gba personen wordt gezocht met de volgende parameters
+      | naam                | waarde                                          |
+      | type                | RaadpleegMetBurgerservicenummer                 |
+      | burgerservicenummer | 000000024                                       |
+      | fields              | burgerservicenummer,verblijfplaats.functieAdres |
+      Dan heeft de response een object met de volgende gegevens
+      | naam     | waarde                                                                  |
+      | type     | https://datatracker.ietf.org/doc/html/rfc7231#section-6.5.3             |
+      | title    | U bent niet geautoriseerd voor één of meerdere opgegeven field waarden. |
+      | status   | 403                                                                     |
+      | detail   | De foutieve fields waarden zijn: fields[1].                             |
+      | code     | authorization                                                           |
+      | instance | /haalcentraal/api/brp/personen                                          |
+
```

Comment:
Hier mist de regel met 'Voorbeelden:'.
Verder is de laatste kolom in deze tabel niet noodzakelijk.

Reply:
toegevoegd aan scenario titel